examples/tasks/task_vlcs.py

```diff
@@ -47,13 +47,13 @@
                        dict_domain2imgroot={
                            "caltech": os.path.join(
                                path_this_file,
-                               "../../data/vlcs_mini/caltech/"),
```

Comment:
this file is not inside the package, so not used by domainlab. It is just an example